### PR TITLE
Update taskcluster-taskgraph to v8.1.0

### PR DESCRIPTION
**Pull request by rvandermeulen**
The changes to requirements.in are expected. Taskgraph is baked into the Decision image now per ahal.

### CODE REVIEW

**Comment by rvandermeulen** [comment]
@ahal Looks like I'm hitting scopes issues on the decision task. Is that something you can help with?

File: taskcluster/requirements.in [hunk 1]
```diff
@@ -1,7 +0,0 @@
-# For instructions on managing dependencies, see:
-# https://taskcluster-taskgraph.readthedocs.io/en/latest/howto/bootstrap-taskgraph.html
-
-arrow
-pyyaml
-taskcluster
```
Comment (ahal) [inline taskcluster/requirements.in:6]: While taskgraph is baked in, these other packages are not. So we still need the requirements files for those.

**Comment by ahal** [comment]
> @ahal Looks like I'm hitting scopes issues on the decision task. Is that something you can help with?

Yeah, I'll fix them in a bit. FYI I also just released Taskgraph 8.1.0, though probably not much that would benefit glean in there

**Review by ahal — CHANGES_REQUESTED** [review]
Requesting changes for requirements file issue.

File: taskcluster/requirements.in (hunk 1 above)
Comment (rvandermeulen) [reply taskcluster/requirements.in:6]: It doesn't appear they're actually used. At least, the decision task runs fine without them!

**Review by ahal — APPROVED** [review]

**Review by badboy — APPROVED** [review]

**Pull request merged**